docs/geps/gep-01.rst

```diff
@@ -86,24 +91,31 @@ in English. For column names, we always allow a pure ASCII option, see the next
 Column names (a.k.a. "variables" in Stata)
 ------------------------------------------
 
-We impose a hard limit of 17 characters for all column names that are part of the API,
-i.e., those that are an input to or an output of GETTSIM's main simulation functions.
+We impose a hard limit of 20 characters for all column names that typically user-facing.
 This is for the benefit of Stata users, who face a strict limit of 32 characters for
 their column names. Furthermore, where developers using other languages may store
 different experiments in different variables, Stata users' only chance to distinguish
 them is to append characters to the column names.
 
-If a column is only present for internal use, it starts with an underscore.
+For the same reason, there is a hard limit of 32 characters for variables that users may
+reasonably request.
 
-Even though not implemented at the time of this writing, we plan to allow users to pass
-in English column names and get English column names back. Similarly, there will be a
-pure ASCII option in German (this should always be fulfilled in the English version). If
-there is demand, we could also support standardised variables like in `EUROMOD
-<https://www.euromod.ac.uk/>`_ or the `CNEF <https://cnef.ehe.osu.edu/data/>`_ standard.
+If a column is only present for internal use, it starts with an underscore and there is
+no restriction on the number of characters. Internal columns should be used sparingly.
 
-It might be useful to append an underscore plus one of {``j``, ``m``, ``w``, ``t``} to
-indicate the time unit.
+Across variations that include the same identifier, this identifier should not be
+changed, even if it leads to long variable names (e.g., ``kinderfreibetrag``,
+``_zu_verst_eink_kein_kinderfreibetrag_tu``). This makes searching for identifiers
+easier and less error-prone.
+
+The default time unit is a year. If a column refers to a different time unit, an
+underscore plus one of {``m``, ``w``, ``t``} will indicate the time unit.
+
+The default unit a column refers to is an individual. In case a household or tax unit is
+the relevant unit, an underscore plus one of {``hh``, ``tu``} will indicate the level of
```

Comment:
Should we rename `tu` to something in German? It is not exactly an institutional feature like the variables names but  time units (`t`) is also in German (or should we change that to `d`?).

Plus, should we state that time unit is appended always before `hh` or `tu` if both are used?

Reply:
> Should we rename tu to something in German? It is not exactly an institutional feature like the variables names but time units (t) is also in German (or should we change that to d?).

The latter, which is also used in the parameters. Good catch! 

> Plus, should we state that time unit is appended always before hh or tu if both are used?

Yip. In fact, it is what the sentence below your quote states :smile: